lgc/builder/InOutBuilder.cpp

```diff
@@ -974,47 +974,43 @@ Value *InOutBuilder::readBuiltIn(bool isOutput, BuiltInKind builtIn, InOutInfo i
 // @returns : gl_Barycoord
 Value *InOutBuilder::normalizeBaryCoord(Value *iJCoord) {
   auto baryType = FixedVectorType::get(getFloatTy(), 3);
-  Value *barycoord = UndefValue::get(baryType);
   auto one = ConstantFP::get(getFloatTy(), 1.0);
   auto zero = ConstantFP::get(getFloatTy(), 0.0);
 
-  auto iCoord = CreateExtractElement(iJCoord, uint64_t(0));
-  auto jCoord = CreateExtractElement(iJCoord, 1);
-  auto kCoord = CreateFSub(ConstantFP::get(getFloatTy(), 1.0), iCoord);
-  kCoord = CreateFSub(kCoord, jCoord);
+  Value *hwCoord[3];
```

Comment:
A nit: hwCoord[3] = {};
If someone changes following codes in the future, there is still an initialized null value.